orchagent/main.cpp

```diff
@@ -486,8 +494,8 @@ int main(int argc, char **argv)
     shared_ptr<ZmqServer> zmq_server = nullptr;
     if (enable_zmq)
     {
-        SWSS_LOG_NOTICE("Instantiate ZMQ server : %s", zmq_server_address.c_str());
-        zmq_server = make_shared<ZmqServer>(zmq_server_address.c_str());
+        SWSS_LOG_NOTICE("Instantiate ZMQ server : %s, %s", zmq_server_address.c_str(), vrf.c_str());
+        zmq_server = make_shared<ZmqServer>(zmq_server_address.c_str(), vrf.c_str());
```

Comment:
Test case failed because latest swss-common lib still not ready:

Oct 10 08:30:24.926763 57bc91e73bd9 NOTICE #orchagent: :- main: Instantiate ZMQ server : tcp://127.0.0.1:8100, 
Oct 10 08:30:24.927125 57bc91e73bd9 INFO #supervisord: orchagent /usr/bin/orchagent: symbol lookup error: /usr/bin/orchagent: undefined symbol: 





_ZN4swss9ZmqServerC1ERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEES8_
Oct 10 08:30:24.930753 57bc91e73bd9 INFO #supervisord 2024-10-10 08:30:24,930 INFO exited: orchagent (exit status 127; not expected)
Oct 10 08:30:25.934284 57bc91e73bd9 INFO #supervisord 2024-10-10 08:30:25,933 INFO spawned: 'orchagent' with pid 665

Reply:
It's because pipeline download lib from 2 different build:

Download from the specified build: #661475
Download from the specified build: #640773